images/macos/provision/core/mongodb.sh

```diff
@@ -4,9 +4,11 @@ source ~/utils/utils.sh
 # MongoDB object-value database
 # Install latest release version of MongoDB Community Edition
 # https://docs.mongodb.com/manual/tutorial/install-mongodb-on-os-x
+toolsetVersion=$(get_toolset_value '.mongodb.version')
+versionToInstall=$(brew search --formulae /mongodb-community@$toolsetVersion/ | awk -F'/' '{print $3}' | tail -1)
```

Comment:
This should be placed after the `brew tap mongodb/brew` for the obvious reason 😃